apps/showcase/src/assets/trainings/sdk/steps/sdk-specs/instructions.md

```diff
@@ -0,0 +1,17 @@
+The first step in the generation of an SDK is the definition of the specifications. These can be written in a file of type JSON or YAML.
+
+The <a href="https://github.com/AmadeusITGroup/otter/tree/main/packages/%40o3r-training/showcase-sdk" target="_blank">Otter Showcase SDK</a> has been created with the
+<a href="https://github.com/swagger-api/swagger-petstore/blob/master/src/main/resources/openapi.yaml" target="_blank">specifications</a> made available by the
+<a href="https://github.com/swagger-api/swagger-petstore" target="_blank">Swagger Petstore</a> repository.
+You can find these specifications in the `openapi.yml` file of the repository.
+
+### Objective
+Before generating your first SDK using an existing specification file, familiarize yourself with the format of the specifications.
```

Comment:
Maybe I'm reading it wrong but it almost sounds like a command `familiarize yourself`
```suggestion
The goal of this exercise is to get a basic understanding of the format of the specifications.
```